### PR TITLE
Feature skeleton

### PR DESCRIPTION
**Pull request by gresendesa**
These modifications are pretty much the same files split into sections. The basic directory structure was built as well.

### CODE REVIEW

File: .idea/uiot.github.io.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ovflowd) [inline .idea/uiot.github.io.iml]: Remove this file, please.

File: Gemfile [hunk 2]
```diff
@@ -3,4 +3,4 @@ source 'https://rubygems.org'
 gem 'jekyll'
 gem 'jekyll-feed'
 gem 'jekyll-admin'
-gem 'jekyll-minifier'
\ No newline at end of file
+#gem 'jekyll-minifier'
```
Comment (ovflowd) [inline Gemfile]: Don't comment those modules.

File: Gemfile.lock [hunk 3]
```diff
@@ -5,11 +5,8 @@ GEM
       public_suffix (>= 2.0.2, < 4.0)
```
Comment (ovflowd) [inline Gemfile.lock]: Don't add the Gemfile.lock to the server. Ignore it on `.gitignore`

**Review by ovflowd — CHANGES_REQUESTED** [review]
Please change those requested changes.

Thanks!

**Comment by gresendesa** [comment]
I'll do it again

**Pull request closed without merging**